Gemfile

```diff
@@ -87,9 +87,10 @@ group :development, :devunicorn, :test do
   gem 'net-ssh', '~> 7.1'
   gem 'net-scp', '>= 4.0.0.rc1'
   gem 'rubocop', '~> 1.50'
-  gem 'rubocop-rspec'
-  gem 'rubocop-rails'
-  gem 'rubocop-performance'
+  gem 'rubocop-capybara', '~> 2.17'
```

Comment:
The version of the gem is being changed to 2.18 as a result of https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/5602